src/routes/media.rs

```diff
@@ -109,3 +138,17 @@ fn process_entries(
         )),
     }
 }
+
+fn get_file_ext(file_fields: &Vec<multipart::server::SavedField>) -> Result<&str, Custom<String>> {
```

Comment:
Is it better to have the error contain a `Custom<String>` or a `Custom<&str>` (where the latter doesn't use the `.to_string()` method of string literals)?

Reply:
Yeah, just tried this out rn. With string refs you need to add a lifetime and tie it to some resource. Better to just move it.

Reply:
Ok makes sense. Would it make sense to use a static lifetime there?

Reply:
I considered that but then `format!` gets messed up.